docker/compose/bootstrap.ini

```diff
@@ -0,0 +1,76 @@
+# this configuration was auto generated with 'sane' defaults
+# change these values as desired
+
+
+[router]
+# number of crypto worker threads
+threads=4
+# path to store signed RC
+contact-file=/root/.lokinet/self.signed
+# path to store transport private key
+transport-privkey=/root/.lokinet/transport.private
+# path to store identity signing key
+ident-privkey=/root/.lokinet/identity.private
+# encryption key for onion routing
+encryption-privkey=/root/.lokinet/encryption.private
+
+# uncomment following line to set router nickname to 'lokinet'
+#nickname=lokinet
+
+
+[logging]
+level=info
+# uncomment for logging to file
+#type=file
+#file=/path/to/logfile
+# uncomment for syslog logging
+#type=syslog
+
+[metrics]
+json-metrics-path=/root/.lokinet/metrics.json
+
+# admin api (disabled by default)
+[api]
+enabled=true
+#authkey=insertpubkey1here
+#authkey=insertpubkey2here
+#authkey=insertpubkey3here
+bind=127.0.0.1:1190
+
+# system settings for privileges and such
+[system]
+user=lokinet
+group=lokinet
+pidfile=/root/.lokinet/lokinet.pid
+
+# dns provider configuration section
+[dns]
+# resolver
+upstream=1.1.1.1
+bind=127.3.2.1:53
+
+# network database settings block
+[netdb]
+# directory for network database skiplist storage
+dir=/netdb
+
+# lokid settings (disabled by default)
+[lokid]
+enabled=false
+jsonrpc=127.0.0.1:22023
+#service-node-seed=/path/to/servicenode/seed
+
+# network settings
+[network]
+profiles=/root/.lokinet/profiles.dat
+enabled=true
+exit=false
+#exit-blacklist=tcp:25
+#exit-whitelist=tcp:*
+#exit-whitelist=udp:*
+ifaddr=10.200.0.1/8
```

Comment:
why `/8`?

Reply:
It’s what’s in https://github.com/loki-project/loki-network/blob/2bdafc66033591e2342d01862ef8b08bfd6c6f4b/lokinet-docker.ini#L77, can probably be tweaked